students/jemielity_kamil/lesson_06_dicts_tuples_sets_args_kwargs/access_rights.py

```diff
@@ -0,0 +1,13 @@
+
+Dictionary = {}
```

Comment:
Dictionary is like any other variable. Please use PEP8 lowercase convention

Reply:
@kamwar fixed